cfg/rh-1.0/policies.yaml

```diff
@@ -212,7 +227,9 @@ groups:
 
       - id: 5.3.2
         text: "Ensure that all Namespaces have Network Policies defined (Manual)"
-        type: "manual"
```

Comment:
I think we should have this as manual, otherwis the command output will be very big. and we dont have test so there is no point to run this command.